spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/interpreter/rules/FillNullsRuleInterpreter.scala

```diff
@@ -44,15 +45,14 @@ case class FillNullsRuleInterpreter(rule: FillNullsConformanceRule) extends Rule
       rule.outputColumn
     )
 
-    val dataType = SchemaUtils.getFieldType(rule.inputColumn, df.schema).get
+    val dataType: DataType = SchemaUtils.getFieldType(rule.inputColumn, df.schema).get
```

Comment:
**Tiny (not worth its on commit unless other changes: ** `val targetDataType` would be an explanatory name, IMO